pygmt_paper_numpy_datetime.ipynb

```diff
@@ -0,0 +1,78 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {
+      "image/png": {
+       "width": 500
+      }
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "import datetime\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import pygmt\n",
+    "\n",
+    "fig = pygmt.Figure()\n",
+    "\n",
+    "fig.basemap(\n",
+    "    projection=\"X12c/6c\",\n",
+    "    region=[datetime.date(2016, 1, 1), datetime.date(2026, 12, 31), 0, 1000],\n",
+    "    frame=[\"x+lyear\", \"y+lGitHub stars\"],\n",
+    ")\n",
+    "\n",
+    "for csvfile, color, label in zip(\n",
+    "    [\"star_history_20251017_gmt.csv\", \"star_history_20251017_pygmt.csv\", \"star_history_20251017_gmtjl.csv\"],\n",
```

Comment:
What about remove the date information from the file names, which can make the code fit in one line.

Reply:
Done in commit https://github.com/GenericMappingTools/pygmt-paper-figures/pull/10/commits/2550708533751170ba82c90586a90aab574e2bf1.